packages/blocks/src/text-input/validations.js

```diff
@@ -8,4 +8,14 @@ export default {
 			return 'Please enter a valid email address.';
```

Comment:
Not part of this PR, but I just noticed this one doesn't have a translation either